### PR TITLE
Update SDK to address fixes of SDK

### PR DESCRIPTION
**Pull request by rahuldutta90**
## Checklist

- [x] I have read the [_Submitting Changes_](https://github.com/Azure/azure-powershell/blob/master/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](https://github.com/Azure/azure-powershell/blob/master/CONTRIBUTING.md)
- [x] The title of the PR is clear and informative
- [x] The appropriate [change log has been updated](https://github.com/Azure/azure-powershell/blob/master/CONTRIBUTING.md#updating-the-change-log)
- [x] The PR does not introduce [breaking changes](https://github.com/Azure/azure-powershell/blob/master/documentation/breaking-changes/breaking-changes-definition.md)
- [x] If applicable, the changes made in the PR have proper test coverage
- [x] For public API changes to cmdlets:
    - [x] the changes have gone through a [cmdlet design review](https://github.com/Azure/azure-powershell-cmdlet-review-pr) (Microsoft internal partners only)
    - [x] the cmdlet markdown files were [generated using the `platyPS` module](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/help-generation.md)

### CODE REVIEW

**Comment by rahuldutta90** [comment]
@cormacpayne  Also can we put this change for the last AzureRm release? Basically it will bring SDK fixes to the azurerm release

File: src/ResourceManager/DataLakeStore/Commands.DataLakeStore/ChangeLog.md [hunk 1]
```diff
@@ -18,7 +18,7 @@
         - Additional information about change #1
 -->
 ## Upcoming Release
-
+* Update the sdk version of dataplane to 1.1.14 to include SDK fixes
```
Comment (markcowl) [inline src/ResourceManager/DataLakeStore/Commands.DataLakeStore/ChangeLog.md]: This is meant to be customer-facing information,  please uodate this  with customer-facing changes that customers will understand

Comment (rahuldutta90) [reply src/ResourceManager/DataLakeStore/Commands.DataLakeStore/ChangeLog.md]: @markcowl I have added the link to our SDKs changelog. That should do right?

File: src/ResourceManager/DataLakeStore/Commands.DataLakeStore/ChangeLog.md [hunk 2]
```diff
@@ -18,7 +18,7 @@
         - Additional information about change #1
 -->
 ## Upcoming Release
-
+* Update the sdk version of dataplane to 1.1.14. Please refer https://github.com/Azure/azure-data-lake-store-net/blob/master/CHANGELOG.md#version-1114 for SDK fixes.
```
Comment (markcowl) [inline src/ResourceManager/DataLakeStore/Commands.DataLakeStore/ChangeLog.md]: No, please summarize these changes here, that is what the changelog is for.

Comment (rahuldutta90) [reply src/ResourceManager/DataLakeStore/Commands.DataLakeStore/ChangeLog.md]: Fixed it. Summarized the fixes.

**Pull request merged**